pkg/trace/obfuscate/sql.go

```diff
@@ -124,6 +152,7 @@ type groupingFilter struct {
 // has been recognized. A grouping is composed by items like:
 //   * '( ?, ?, ? )'
 //   * '( ?, ? ), ( ?, ? )'
+//   * 'ARRAY [ ?, ?, ?]'
```

Comment:
A space is missing or is this correct?
```suggestion
//   * 'ARRAY [ ?, ?, ? ]'
```